android/app-example/src/main/java/com/badoo/ribs/example/rib/lorem_ipsum/LoremIpsum.kt

```diff
@@ -0,0 +1,24 @@
+package com.badoo.ribs.example.rib.lorem_ipsum
+
+import com.badoo.ribs.core.Rib
+import com.badoo.ribs.core.directory.inflateOnDemand
+import com.badoo.ribs.core.view.ViewFactory
+import com.badoo.ribs.example.R
+import io.reactivex.functions.Consumer
+
+interface LoremIpsum : Rib {
```

Comment:
If we gonna use that code as a guide book, best practice for newcomers, we have to think about the proper real app.
Something simple and light, but without LoremIpsum, Foo, AnotherRib, RandomStuff and so on.

We are creating more and more code, logic becomes complex.
Let's think about it and discuss.

@lukaville

Reply:
I agree - in the long run. For now, I create examples that are the quickest to demonstrate a change to you guys. When APIs and functionality are stable, I'll spend time on creating proper examples, documentation, with proper names, but not before that.